scripts/install.sh

```diff
@@ -69,7 +69,7 @@ platform=
 os=$(uname -s)
 if [ "${os}" = "Linux" ]
 then
-    if [ -n "$(rpm -qa)" ] 2>/dev/null
+    if check rpm && [ -n "$(rpm -qa 2>/dev/null)" ]
```

Comment:
@tobim I haven't checked whether the redirect into the `rpm` call causes issues. Please confirm and resolve if that seems okay.

Reply:
I verified that this silences the unintended output.